src/view/abstracttexteditor.cpp

```diff
@@ -1062,6 +1066,109 @@ void AbstractTextEditor::onKeyboardStateChanged(QString state) {
     d->keyboardState = state;
 }
 
+void AbstractTextEditor::sendKeySequence(const QString &action, const QKeySequence &sequence) {
+
+    // if (debug) qDebug() << InputContextName << __PRETTY_FUNCTION__ << "action" << action;
```

Comment:
This should either be left in with a way to set the debug attribute or left out.

Reply:
Yep, I think I should delete that comment.....I might be a victim of being a "Copy-paste programmer" :grinning:

Reply:
I've deleted the comment in the latest commit.